packages/block-editor/src/hooks/utils.js

```diff
@@ -135,7 +135,20 @@ export function shouldSkipSerialization(
 
 const pendingStyleOverrides = new WeakMap();
 
-export function useStyleOverride( {
+/**
+ * Override a block editor settings style. Leave the ID blank to create a new
+ * style.
+ *
+ * @param {Object} $0     Named parameters.
+ * @param {string} $0.id  Id of the style override, leave blank to create a new
+ *                        style.
+ * @param {string} $0.css CSS to apply.
```

Comment:
Tiny nit: should this be an optional `string` param?

```suggestion
 * @param {Object}  $0     Named parameters.
 * @param {?string} $0.id  Id of the style override, leave blank to create a new
 *                        style.
 * @param {string}  $0.css CSS to apply.
```

Reply:
If we're flagging `id` as optional here, we should probably also then do the same in the [readme](https://github.com/WordPress/gutenberg/pull/63656/files#diff-dd80d95d10df70d186b67b89836b923443e04bb1fcb0f3348773cf69511b5b53R1043).

Reply:
I think JSDocs also supports a better style than `$0` for object params.

Reply:
@Mamaduka Curious what you're referring to. Do you mean a separate type definition?

Reply:
```
 * @param {Object} override
 * @param {string} override.id
 * @param {string} override.css
```